drivers/at86rf2xx/include/at86rf2xx_registers.h

```diff
@@ -330,6 +335,23 @@ extern "C" {
 /** @} */
 
 /**
+ * @name    Bitfield definitions for the XAH_CTRL_2 register
+ *
+ * This register contains both the CSMA-CA retry counter and the frame retry
+ * counter. At this moment only the at86rf232 and the at86rf233 support this
+ * register.
+ *
+ * @{
+ */
+#if defined(AT86RF2XX_HAVE_RETRIES) || defined(DOXYGEN)
```

Comment:
defined(DOXYGEN) shouldn't really be needed since the flag is already defined if DOXYGEN is defined.

Reply:
Good one, will fix in the next push